src/Workspaces/CSharp/Portable/Extensions/SemanticModelExtensions.cs

```diff
@@ -218,7 +218,10 @@ public static ImmutableArray<ParameterName> GenerateNames(IList<string> reserved
                         var parameter = member.Parameters[index];
                         if (parameter.Type.OriginalDefinition.TypeKind != TypeKind.TypeParameter)
                         {
-                            return parameter.Name;
+                            if (parameter.Name != "_")
```

Comment:
Is there a better way to determine if the parameter name is a discard?

Reply:
Looks like this is the pattern used elsewhere in IDE
[https://sourceroslyn.io/#Microsoft.CodeAnalysis.CSharp.Workspaces/Classification/SyntaxClassification/DiscardSyntaxClassifier.cs,04c828b42766b963](https://sourceroslyn.io/#Microsoft.CodeAnalysis.CSharp.Workspaces/Classification/SyntaxClassification/DiscardSyntaxClassifier.cs,04c828b42766b963)

Reply:
Found this
[https://sourceroslyn.io/#Microsoft.CodeAnalysis.CSharp/Syntax/SyntaxKindFacts.cs,d66fbdb93ecd0782,references](https://sourceroslyn.io/#Microsoft.CodeAnalysis.CSharp/Syntax/SyntaxKindFacts.cs,d66fbdb93ecd0782,references)